CONTRIBUTING.md

```diff
@@ -0,0 +1,247 @@
+
+```md
+- [ ] CHANGELOG.md updated
+- [ ] Rebased/mergable
+- [ ] Tests pass
+- [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
+```
+
+Security Vulnerability Reporting
+---------------
+InfluxData takes security and our users' trust very seriously. If you believe you have found a security issue in any of our
+open source projects, please responsibly disclose it by contacting security@influxdata.com. More details about 
+security vulnerability reporting, 
+including our GPG key, [can be found here](https://www.influxdata.com/how-to-report-security-vulnerabilities/).
+
+
+Signing the CLA
+---------------
+
+If you are going to be contributing back to InfluxDB please take a
+second to sign our CLA, which can be found
+[on our website](https://influxdata.com/community/cla/).
+
+Installing Go
+-------------
+InfluxDB requires Go 1.12.
+
+At InfluxData we find gvm, a Go version manager, useful for installing Go. For instructions
+on how to install it see [the gvm page on github](https://github.com/moovweb/gvm).
+
+After installing gvm you can install and set the default go version by
+running the following:
+
+    gvm install go1.11
+    gvm use go1.12 --default
+
+Revision Control Systems
+-------------
+Go has the ability to import remote packages via revision control systems with the `go get` command.  To ensure that you can retrieve any remote package, be sure to install the following rcs software to your system.
+Currently the project only depends on `git` and `mercurial`.
+
+* [Install Git](http://git-scm.com/book/en/Getting-Started-Installing-Git)
+* [Install Mercurial](http://mercurial.selenic.com/wiki/Download)
+
+Getting the source
+------
+Setup the project structure and fetch the repo like so:
+
+```bash
+    cd $GOPATH/src
+    git clone git@github.com:influxdata/influxdb.git
+    cd influxdb
+    export  GO111MODULE=on
+    go mod vendor
+    go install ./...
+    go test ./...
+```
+
+You can add the line `export GOPATH=$HOME/gocodez` to your bash/zsh file to be set for every shell instead of having to manually run it everytime.
+
+Cloning a fork
+-------------
+If you wish to work with fork of InfluxDB, your own fork for example, you must still follow the directory structure above. But instead of cloning the main repo, instead clone your fork. Follow the steps below to work with a fork:
+
+```bash
+    export GOPATH=$HOME/gocodez
+    mkdir -p $GOPATH/src/github.com/influxdata
+    cd $GOPATH/src/github.com/influxdata
+    git clone git@github.com:<username>/influxdb
+```
+
+Retaining the directory structure `$GOPATH/src/github.com/influxdata` is necessary so that Go imports work correctly.
+
+Build and Test
+-----
+
+Make sure you have Go installed and the project structure as shown above. To then get the dependencies for the project, execute the following commands:
+
+```bash
+cd $GOPATH/src/github.com/influxdata/influxdb
+dep ensure
```

Comment:
This isn't correct. I think you should document how to build the project using `make`, which would then ensure the UI and so on is built.